metrics-scraper/monitoring.go

```diff
@@ -69,16 +70,49 @@ func Start(configPath string) {
 	wg.Wait()
 }
 
+// tryConnect, try to connect to the node forever until it works
+// it'll stop when it acquire a connection or it's asked to stop
+func tryConnect(
```

Comment:
What are the values of the gauges if a node is down and we keep trying to connect?

Reply:
they'll be empty

Reply:
addressed in: d1755782d3115b584b67e6fb763dc3d62386ef81